src/FFmpegReader.cpp

```diff
@@ -977,7 +982,7 @@ std::shared_ptr<Frame> FFmpegReader::ReadStream(int64_t requested_frame) {
 
 		// Video packet
 		if ((info.has_video && packet && packet->stream_index == videoStream) ||
-			(info.has_video && !packet && packet_status.video_decoded < packet_status.video_read) ||
+			(info.has_video && packet_status.video_decoded < packet_status.video_read) ||
```

Comment:
Call `ProcessVideoPacket()` anytime `video_decoded` < `video_read`... so we can potentially resend a packet if needed, or try and retrieve a processed AVFrame from the decoder.